docs/udev-configuration.md

```diff
@@ -27,7 +27,7 @@ sound devices made by the vendor `Great Vendor`. You set the following udev rule
 Configuration:
 ```yaml
 spec:
```

Comment:
not sure if this has been updated yet, but is it possible that this could work:
```yaml
discoveryHandler:
  name: udev
  discoveryDetails:
    udevRules:
    - '...'
```

Reply:
switched to string discovery_details in 8698b3197b9

Reply:
Still looks like it needs updating.

Reply:
Fixed in 0a1fd2556

Reply:
do we need `name: udev` here?